pkg/model/manifest.go

```diff
@@ -1020,6 +1021,70 @@ func (d *Dependency) GetTimeout(defaultTimeout time.Duration) time.Duration {
 	return defaultTimeout
 }
 
+// ExpandVars sets dependencies values if values fits with list params
+func (d *Dependency) ExpandVars(variables []string) error {
+	parser := parse.New("string", append(os.Environ(), variables...), &parse.Restrictions{})
+
+	expandedBranch, err := parser.Parse(d.Branch)
+	if err != nil {
+		return err
```

Comment:
should we wrap the error mentioning which field is the one failing? This way it will be easier to debug:
```suggestion
		return fmt.Errorf("error expaning 'branch': %w", err)
```

Reply:
done 027299b5b84341aa1a4f0eb8b6ce2eea2cc007e7